### PR TITLE
Kadane's Algorithm in C++

### PR DESCRIPTION
**Pull request by JanhaviSoni**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added Kadane's Algorithm written in C++ in DP folder

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#3436

### CODE REVIEW

File: C-Plus-Plus/dp/Kadane_Algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	int arr[arr_len];
+	for (int ind = 0; ind < arr_len; ind++)
+		cin >> arr[ind];
+	cout << kadane_1(arr, arr_len);
+	cout << '\n';
+	cout << kadane_2(arr, arr_len);
+	cout << '\n';
+
+	return 0;
+}
```
Comment (gat786) [inline C-Plus-Plus/dp/Kadane_Algorithm.cpp]: add a line break at the End of File

**Review by gat786 — CHANGES_REQUESTED** [review]

**Comment by gat786** [comment]
The code looks good to me. Theres just one slight change you need to make before we merge it.
Thanks for contributing :-)

**Comment by JanhaviSoni** [comment]
@gat786 Please check

**Review by gat786 — APPROVED** [review]

**Comment by gat786** [comment]
Great work 😁 Have a great day 👍

**Comment by ankitaggarwal23** [comment]
@all-contributors bot please add @JanhaviSoni for code

**Pull request merged**